pyvista/utilities/reader.py

```diff
@@ -1248,6 +1249,30 @@ class Plot3DMetaReader(BaseReader):
     _class_reader = staticmethod(_vtk.lazy_vtkPlot3DMetaReader)
 
 
+class Plot3DFunctionEnum(enum.IntEnum):
```

Comment:
Does this need to be added to the documentation somehow?  I think it will be missing, but I haven't built the docs to check for this PR.

Reply:
Just in case you're unaware, you can download the build artifact on successful CI doc builds via the checks tab -> build documentation (on the left) -> docs-build (under Artifacts), in this case https://github.com/pyvista/pyvista/suites/7764071910/artifacts/326430707